tests/dags/providers/provider_api_scripts/test_nypl.py

```diff
@@ -1,136 +1,98 @@
 import json
 import logging
-import os
-from unittest.mock import MagicMock, patch
-
-import requests
-from providers.provider_api_scripts import nypl as np
-
-
-RESOURCES = os.path.join(os.path.abspath(os.path.dirname(__file__)), "resources/nypl")
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+from common.licenses import LicenseInfo
+from common.loader import provider_details as prov
+from common.storage.image import ImageStore
+from providers.provider_api_scripts.nypl import NyplDataIngester
+
+
+RESOURCES = Path(__file__).parent / "resources/nypl"
+CC0 = LicenseInfo(
+    license="cc0",
+    version="1.0",
+    url="https://creativecommons.org/publicdomain/zero/1.0/",
+    raw_url="https://creativecommons.org/publicdomain/zero/1.0/",
+)
 
 logging.basicConfig(
     format="%(asctime)s - %(name)s - %(levelname)s:  %(message)s", level=logging.DEBUG
 )
 
 
+@pytest.fixture(autouse=True)
+def validate_url_string():
+    with patch("common.urls.rewrite_redirected_url") as mock_validate_url_string:
+        mock_validate_url_string.side_effect = lambda x: x
+        yield
+
+
+nypl = NyplDataIngester()
+image_store = ImageStore(provider=prov.NYPL_DEFAULT_PROVIDER)
+nypl.media_stores = {"image": image_store}
```

Comment:
Rather than initializing a new media store, maybe we could use the one that comes with the ingester?
```suggestion
image_store = nypl.media_stores["image"]
```